text/pokedex.asm

```diff
@@ -1,1510 +1,1466 @@
 _RhydonDexEntry::
-	text "Protected by an"
-	next "armor-like hide,"
-	next "it is capable of"
+	text "מוגן על ידי"
+	next "עור דמוי-שריון,"
+	next "מסוגל לחיות"
 
-	page "living in molten"
-	next "lava of 3,600"
-	next "degrees"
+	page "בלבה מותכת"
+	next "בטמפרטורה של"
+	next "006,3 מעלות"
 	dex
 
 _KangaskhanDexEntry::
-	text "The infant rarely"
-	next "ventures out of"
-	next "its mother's"
+	text "הגור בקושי"
+	next "מגיח החוצה"
+	next "מכיסה המגונן"
 
-	page "protective pouch"
-	next "until it is 3"
-	next "years old"
+	page "של אימו"
+	next "עד שהוא"
+	next "בן 3"
 	dex
 
 _NidoranMDexEntry::
-	text "Stiffens its ears"
-	next "to sense danger."
-	next "The larger its"
+	text "מקשיח את אוזניו"
+	next "כדי לזהות סכנה."
+	next "ככל שקרניו"
 
-	page "horns, the more"
-	next "powerful its"
-	next "secreted venom"
+	page "גדולות יותר, כך"
+	next "עוצמתי יותר"
+	next "הארס המופרש מהן"
 	dex
 
 _ClefairyDexEntry::
-	text "Its magical and"
-	next "cute appeal has"
-	next "many admirers."
+	text "בעל קסם אישי"
+	next "מתוק וכובש, לו"
+	next "יש מעריצים רבים."
 
-	page "It is rare and"
-	next "found only in"
-	next "certain areas"
+	page "הוא נדיר וניתן"
+	next "למצוא אותו רק"
+	next "באזורים מסוימים"
 	dex
 
 _SpearowDexEntry::
-	text "Eats bugs in"
-	next "grassy areas. It"
-	next "has to flap its"
+	text "אוכל חרקים"
+	next "באזורים עשביים."
+	next "מוכרח לנפנף"
 
-	page "short wings at"
-	next "high speed to"
-	next "stay airborne"
+	page "במהירות בכנפיו"
+	next "הקצרות כדי"
+	next "להישאר באוויר"
 	dex
 
 _VoltorbDexEntry::
-	text "Usually found in"
-	next "power plants."
-	next "Easily mistaken"
+	text "בדרך כלל נמצאים"
+	next "בתחנות כוח."
+	next "קל להתבלבל בינם"
 
-	page "for a # BALL,"
-	next "they have zapped"
-	next "many people"
+	page "ובין פוכדורים,"
+	next "הם חישמלו"
+	next "הרבה אנשים"
 	dex
 
 _NidokingDexEntry::
-	text "It uses its"
-	next "powerful tail in"
-	next "battle to smash,"
+	text "משתמש בזנבו"
+	next "העוצמתי בקרב"
+	next "לריסוק, לכבילת"
 
-	page "constrict, then"
-	next "break the prey's"
-	next "bones"
+	page "ולשבירת עצמותיו"
+	next "של הטרף"
 	dex
 
 _SlowbroDexEntry::
-	text "The SHELLDER that"
-	next "is latched onto"
-	next "SLOWPOKE's tail"
+	text "הקונכך שנצמד"
+	next "לזנבו של כיסלאט"
+	next "לכאורה ניזון"
 
-	page "is said to feed"
-	next "on the host's left"
-	next "over scraps"
+	page "משאריות האוכל"
+	next "של הפונדקאי שלו"
 	dex
 
 _IvysaurDexEntry::
-	text "When the bulb on"
-	next "its back grows"
-	next "large, it appears"
+	text "כשהפקעת על"
+	next "גבו גדלה,"
+	next "נדמה שהוא מאבד"
 
-	page "to lose the"
-	next "ability to stand"
-	next "on its hind legs"
+	page "את היכולת"
+	next "לעמוד על"
+	next "רגליו האחוריות"
 	dex
 
 _ExeggutorDexEntry::
-	text "Legend has it that"
-	next "on rare occasions,"
-	next "one of its heads"
+	text "האגדה מספרת"
+	next "שבמקרים נדירים,"
```

Comment:
אולי "מספרים שבמקרים"? באנגלית Legend has it לא מתיחס לשום אגדה ספציפית, ובעברית זה פחות ככה.